src/pages/balances/[friendId].tsx

```diff
@@ -55,10 +56,20 @@ const FriendPage: NextPageWithUser = ({ user }) => {
           </div>
         }
         actions={
-          <DeleteFriend
-            friendId={_friendId}
-            disabled={!(youLent?.length === 0 && youOwe?.length === 0)}
-          />
+          <div className="flex items-center space-x-2">
```

Comment:
it will be nice to have the export option here

<img width="755" alt="Screenshot 2024-10-14 at 4 43 28 pm" src="https://github.com/user-attachments/assets/e4ef6c21-2697-40cf-acac-87b9c6d887f6">

Reply:
like this ?

![image](https://github.com/user-attachments/assets/2a33ea17-ffcc-4aa3-91a7-cb0bbacf7e2c)

Reply:
exactly